src/main/webapp/WEB-INF/view/admin.jsp

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Admin</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+  <nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/conversations">Conversations</a>
+    <a href="/about.jsp">About</a>
+  </nav>
```

Comment:
This navigation bar is shared among many pages, so there should be a way to describe it once and then just include everywhere else.

No need to fix this in this PR, as this is not it's purpose, but maybe create new issue as a reminder to look at this at some point?

Reply:
Great idea, sounds good.